moped-database/views/combined_project_funding_view.sql

```diff
@@ -45,8 +45,9 @@ SELECT
     ecapris_subproject_funding.ecapris_subproject_id,
     TRUE AS is_synced_from_ecapris
 FROM ecapris_subproject_funding
+JOIN moped_project ON ecapris_subproject_funding.ecapris_subproject_id = moped_project.ecapris_subproject_id
 WHERE NOT (EXISTS (
         SELECT 1
-        FROM moped_proj_funding
-        WHERE moped_proj_funding.fdu = ecapris_subproject_funding.fdu AND moped_proj_funding.is_deleted = FALSE
+        FROM moped_proj_funding mpf
+        WHERE mpf.fdu = ecapris_subproject_funding.fdu AND mpf.project_id = moped_project.project_id AND mpf.is_deleted = FALSE
     ));
```

Comment:
This change makes the view produce eCAPRIS rows for every project based on its eCAPRIS subproject ID. Since more than one Moped project can have the same eCAPRIS subproject ID, we need to think about de-duplicating by existing FDUs on a per-project basis.